src/profile-logic/process-profile.js

```diff
@@ -818,7 +818,11 @@ function _adjustMarkerTimestamps(
  * Throws an exception if it encounters an incompatible profile.
  * For a description of the processed format, look at docs/gecko-profile-format.md
  */
-export function processProfile(geckoProfile: GeckoProfile): Profile {
+export function processProfile(rawProfile: Object): Profile {
```

Comment:
maybe type it like this: `GeckoProfile | { profile: GeckoProfile }` ?

Reply:
Ohhhh, nice.